src/Symfony/Component/Security/Http/composer.json

```diff
@@ -31,7 +31,8 @@
     },
     "conflict": {
         "symfony/event-dispatcher": "<4.3",
-        "symfony/security-csrf": "<4.4"
+        "symfony/security-csrf": "<4.4",
+        "symfony/security-bundle": "<5.2"
```

Comment:
Have added this because both `symfony/security` and `symfony/security-bundle` need to be installed in the "=>5.2" version to work properly together.

Reply:
`security-http` should not be aware of `security-bundle`, only security-bundle knows about security components and has version constraints for them

Reply:
Hmm, I'm wondering how would you ensure compatibility then? Combining a 5.1 `symfony/security-bundle` with 5.2 `symfony/security` would lead to an odd behavior.